### ISSUE TITLE
Get-WinSCPHostKeyFingerprint documentation updates

### ISSUE THREAD

**Issue by jeffreymcclain** [dotps1/WinSCP#91]
### Issue Description

'Algorithm' parameter is required with the latest version of the module, so the documentation should reflect this. 

https://github.com/dotps1/WinSCP/wiki/Get-WinSCPHostKeyFingerprint#example-1
https://github.com/dotps1/WinSCP/wiki/Get-WinSCPHostKeyFingerprint#-algorithm

### WinSCP-PowerShell Version

5.13.8

**Comment by dotps1** [comment]
maybe make a pull request for this, rather them me fixing a typo?

**Comment by dotps1** [comment]
the `WinSCP-Help.xml` is updated, but I will not be pushing a new version to the PowerShell gallery for this change, you can download the file from here and update in your local module repo.  else the change will be released with the next version of WinSCP, or if there is bug fix that needs to be pushed.  thanks.

**Comment by jeffreymcclain** [comment]
@dotps1 I apologize for not just submitting a pull request to save you time, it was the end of the day Friday and I happened to notice those minor issues right before leaving work, so I figured Id report them before I forgot. I was planning to submit a follow up pull request on Monday referencing the issue, but I see you've already fixed it. Thanks again, I appreciate your fast response time and fixes!
